.github/renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "automerge": false,
+  "baseBranches": [
+    "main"
+  ],
+  "branchPrefix": "renovate-",
+  "dependencyDashboard": false,
+  "extends": [
+    "config:recommended"
+  ],
+  "hashedBranchLength": 24,
+  "labels": [
+    "dependency-update"
+  ],
+  "major": {
+    "automerge": false,
+    "labels": [
+      "dependency-update",
+      "major-dependency-update"
+    ]
+  },
+  "prHourlyLimit": 10,
+  "rebaseWhen": "behind-base-branch",
+  "reviewersSampleSize": 1,
+  "schedule": [
+    "at any time"
+  ],
+  "timezone": "America/New_York",
```

Comment:
why NY timezone?

Reply:
The instance of Renovatbot for our team in CHI cluster is running in region europe-west1. 

This is same for standard, and used by other team as well, I don't think it has any impact, we are good